ambf_ros_modules/ambf_comm/scripts/ambf_rigid_body.py

```diff
@@ -130,7 +130,7 @@ def get_joint_pos(self, joint_name_or_idx):
         else:
             return None
 
-    def get_joint_vel(self, joint_name_or_idx):
+    def get_joint_velocity(self, joint_name_or_idx):
```

Comment:
Did we agree to get_joint_vel or get_joint_velocity? I think the former, right? Probably would want to fix that